src/Rings/MPolyQuo.jl

```diff
@@ -1544,56 +1544,45 @@ end
     minimal_generating_set(I::MPolyQuoIdeal{<:MPolyDecRingElem})
 
 Given a homogeneous ideal `a` of a graded affine algebra over a field,
-return an array containing a minimal set of generators of `a`.
+return an array containing a minimal set of generators of `a`. If `I`
+is the zero ideal an empty list is returned.
 
 # Examples
 ```jldoctest
 julia> R, (x, y, z) = graded_polynomial_ring(QQ, ["x", "y", "z"]);
 
-julia> V = [x, z^2, x^3+y^3, y^4, y*z^5];
-
-julia> I = ideal(R, V)
-ideal(x, z^2, x^3 + y^3, y^4, y*z^5)
-
 julia> A, p = quo(R, ideal(R, [x-y]));
 
-julia> a = ideal(A, [p(x) for x in V]);
+julia> V = [x, z^2, x^3+y^3, y^4, y*z^5];
+
+julia> a = ideal(A, V);
 
 julia> minimal_generating_set(a)
 2-element Vector{MPolyQuoRingElem{MPolyDecRingElem{QQFieldElem, QQMPolyRingElem}}}:
- x
+ y
  z^2
 ```
 """
-function minimal_generating_set(I::MPolyQuoIdeal{<:MPolyDecRingElem}; ordering::MonomialOrdering = default_ordering(base_ring(base_ring(I))))
+function minimal_generating_set(I::MPolyQuoIdeal{<:MPolyDecRingElem})
   # This only works / makes sense for homogeneous ideals. So far ideals in an
```

Comment:
Stupid question (not to the code of this PR, but to the minimal generating set): 
The notion also makes sense in a local ring as emulated in Singular by a choice of a local ordering for the localization at the origin. Do we envision to also support the local case in the future?